homeassistant/config.py

```diff
@@ -116,6 +117,19 @@
 """
 
 
+@dataclass
+class ConfigExceptionInfo:
+    """Configuration exception info class."""
+
+    ex: Exception
+    translation_key: str
+    platform_name: str
+    config: ConfigType
+    log_message: str | None = None
```

Comment:
```suggestion
```

Reply:
For this PR I suggest not to remove this yet as it is needed to log the messages. We could keep the logging for unexpected errors as we have it now, in that case log message can be removed. Is that a good solution?

Reply:
I'll create a mapping on the translation key for the errors that do not have a generated error message.